playground/backend/internal/setup_tools/life_cycle/life_cycle_setuper.go

```diff
@@ -143,3 +152,41 @@ func updateJavaLogConfigFile(paths fs_tool.LifeCyclePaths) error {
 	}
 	return nil
 }
+
+func prepareSbtFiles(lc *fs_tool.LifeCycle, pipelineFolder string, workingDir string) (*fs_tool.LifeCycle, error) {
+	cmd := exec.Command("sh", filepath.Join(workingDir, "new_scio_project.sh"))
+	cmd.Dir = pipelineFolder
+	_, err := cmd.Output()
+	if err != nil {
+		return nil, err
+	}
+
+	sourceFileFolder := filepath.Join(pipelineFolder, projectPath)
+	fileName := lc.Paths.SourceFileName
+	absFileFolderPath, _ := filepath.Abs(sourceFileFolder)
+	absFilePath, _ := filepath.Abs(filepath.Join(absFileFolderPath, fileName))
+	absLogFilePath, _ := filepath.Abs(filepath.Join(absFileFolderPath, logFileName))
+	projectFolder, _ := filepath.Abs(filepath.Join(pipelineFolder, scioProjectName))
+	executableName := lc.Paths.ExecutableName
+
+	_, err = exec.Command("rm", filepath.Join(absFileFolderPath, "WordCount.scala")).Output()
```

Comment:
Why is this thing hardcoded? Did you decide to delete this part per the conversation below?

Reply:
There is no need anywhere else for this constant, but I can move it to constant variab
e

Reply:
Yes, please